pkg/cmd/admin.go

```diff
@@ -55,11 +56,25 @@ func newAdminDataCommand() *cobra.Command {
 			return nil
 		},
 	}
-	v := version.Get()
+
+	defaultBackupName := ""
+	if dataVer, err := prerun.GetVersionOfData(); err == nil {
```

Comment:
Huh, interesting. `GetVersionOfData()` returns unexported struct, but it works and linter is happy...